src/Tribe/Query.php

```diff
@@ -712,8 +704,8 @@ public static function getHideFromUpcomingEvents() {
 		 * @return array The counts array.
 		 */
 		public static function getEventCounts( $args = array() ) {
+			_deprecated_function( __FUNCTION__, '3.11' );
```

Comment:
There is actually no replacement for this function right now.. it returns an array of days of the month and the IDs & number of events on each day. Currently not doing that anywhere anymore.